backend/src/main/java/com/praksa/auction/repository/ProductRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.praksa.auction.repository;
+
+import com.praksa.auction.model.Product;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.Date;
+import java.util.List;
+
+public interface ProductRepository extends JpaRepository<Product, Long> {
+    @Query("SELECT p FROM Product p ORDER BY RANDOM()")
```

Comment:
Since we are using native query here, in order to avoid warning for it, it is good to add its flag like this:
`@Query(value = "SELECT p FROM Product p ORDER BY RANDOM()", nativeQuery = true)`